### ISSUE TITLE
Error from Copyright Info Dashboard

### ISSUE THREAD

**Issue by mjherzog** [aboutcode-org/scancode-workbench#597]
### Description
I loaded a -clipeu Development Codebase Scan and selected the Copyright Info Dashboard for the whole codebase - 14572 files and 1516 directories. The display just shows the progress wheels. In Developer Tools I see the error:

```
caught (in promise) TypeError: Cannot read properties of null (reading 'toString')
    at pie.ts:47:23
    at Array.forEach (<anonymous>)
    at pG (pie.ts:44:9)
    at CopyrightInfoDash.tsx:70:11
```

### System configuration
- SCWB v4.0.0rc4 running on Windows 10
- Scan from SCTK v32.0.6

**Comment by OmkarPh** [comment]
Thanks for reporting @mjherzog 

I've recently refactored this code while writing the tests in [feature/unittests](https://github.com/nexB/scancode-workbench/tree/feature/unittests) and this issue(and similar ones occurring due to null-ish values received from sqlite) should be fixed by this

**Comment by OmkarPh** [comment]
Closing as it is fixed and available in [v4.0.0 Release](https://github.com/nexB/scancode-workbench/releases/tag/v4.0.0)
